Source/Codecov/Factories/ContinuousIntegrationServerFactory.cs

```diff
@@ -7,7 +7,7 @@ internal static class ContinuousIntegrationServerFactory
     {
         public static IContinuousIntegrationServer Create()
         {
-            var continuousIntegrationServers = new IContinuousIntegrationServer[] { new AppVeyor(), new Travis(), new TeamCity(), new AzurePipelines() };
+            var continuousIntegrationServers = new IContinuousIntegrationServer[] { new AppVeyor(), new Travis(), new TeamCity(), new AzurePipelines(), new Jenkins() };
```

Comment:
I really should look into adding these dynamically instead of hard-coding them every time a new CI is implemented.

Oh well, a task for another day :smile: